docs/api/dialog.md

```diff
@@ -218,6 +233,11 @@ The `filters` specifies an array of file types that can be displayed, see
 **Note:** On macOS, using the asynchronous version is recommended to avoid issues when
 expanding and collapsing the dialog.
 
+**Note:** On Linux `defaultPath` is not supported when using portal file chooser
```

Comment:
`defaultPath` is only unsupported on versions <4 for open dialogs - save dialogs work as expected.